spec/std/time/time_spec.cr

```diff
@@ -34,15 +34,15 @@ describe Time do
     time.nanosecond.should eq(999_999_999)
   end
 
-  it "initialize millisecond negative" do
+  it "initialize nanosecond negative" do
     Time.expect_invalid do
-      Time.new(9999, 12, 31, 23, 59, 59, -1)
+      Time.new(9999, 12, 31, 23, 59, 59, nanosecond: -1)
     end
   end
 
   it "initialize nanoseconds 1_000_000_000" do
```

Comment:
ditto, with `fail initialize when nanosecond is a second` or something like that